utils/eventListeners.ts

```diff
@@ -114,15 +114,20 @@ export async function waitSudoOperationSuccess(
 
 export async function waitSudoOperationFail(
   checkingEvent: MangataGenericEvent[],
-  expectedError: string
+  expectedError: string,
+  anotherExpectedError = expectedError
 ) {
   const filterBootstrapEvent = checkingEvent.filter(
     (extrinsicResult) => extrinsicResult.method === "Sudid"
   );
 
   const BootstrapError = await getEventErrorFromSudo(filterBootstrapEvent);
 
-  expect(BootstrapError.method).toContain(expectedError);
+  const errorComparison =
+    BootstrapError.method === expectedError ||
+    BootstrapError.method === anotherExpectedError;
+
+  expect(errorComparison).toEqual(true);
```

Comment:
plz  create a list of errors and check if the ones returned exist on that list. 
https://stackoverflow.com/questions/66032487/jest-expect-to-array-contains-element

The reason for this is that when this fail, instead of ( expected false to be true ) will be something like: expected to be one of { err1, err2} but found err

Reply:
done